Core/SymbolValidation/SymbolValidator.cs

```diff
@@ -290,7 +297,7 @@ private async Task<SymbolValidatorResult> CalculateValidity(IReadOnlyList<IFile>
 #else
                             using var getStream = await response.Content!.ReadAsStreamAsync().ConfigureAwait(false);
 #endif
-                            using var tempFile = new TemporaryFile(getStream, ".npe.snupkg");
+                            using var tempFile = GetTemporaryFile(getStream, extension: ".npe.snupkg", name: fileName);
```

Comment:
Is there any reason to put `.npe.` in the filename instead of just using the original file name?

Reply:
I didn't want to run until long path issues. `.npe.nupkg` is less likely than `{long ID}.{long version}.snupkg`. Additionally, the .snupkg temp file was never triggering the alerts. It was the assembly temp file case.